github-release.js

```diff
@@ -0,0 +1,51 @@
+const { Octokit } = require('@octokit/rest');
+const fs = require('fs');
+
+const accessToken = process.env.GITHUB_TOKEN;
+const appVersion = `v${process.env.npm_package_version}`;
+const buildNamePrefix = `Code.org Maker App-${process.env.npm_package_version}-`;
+const buildNameSuffixes = ['mac.dmg', 'win.exe', 'linux.deb', 'linux.AppImage'];
+
+if (accessToken) {
+  (async() => {
+    const octokit = new Octokit({
+      auth: accessToken,
+    });
+    const release = await createRelease();
+    const releaseId = release.data.id;
+
+    await uploadAssets(releaseId);
+    console.log(`Release drafted. Visit ${release.data.html_url} to publish.`);
```

Comment:
should we automatically publish the release at this step (using `octokit.rest.repos.updateRelease` to set `draft` to false)? 

if there's anything we want the deploying engineer to double-check, i don't think we should automatically publish. and, in that case, we should add those steps to this log (e.g., "make sure ${buildNameSuffixes} have all been uploaded")

Reply:
We want the engineer to add release notes before publishing. I will update the wording here 👍

Reply:
...updated to include a nudge to add release notes